src/main/java/com/epam/prd/jjop/helpers/mutablelist/Main.java

```diff
@@ -0,0 +1,173 @@
+package com.epam.prd.jjop.helpers.mutablelist;
+
+import java.util.*;
+
+/**
+ * @author Mariusz Bal
```

Comment:
Add here one, short sentence about what this is class about. eg.

`Shows how to have an IM/MUTABLE list with Arrays.asList, Collections.list  and List.of.`